README.md

```diff
@@ -114,7 +117,7 @@ To debug the examples using JTAG and GDB, follow these steps:
 
 ## Memory layout
 
-The following diagram illustrates the run-time memory layout and binary image layout when direct boot is used.
+The following diagram illustrates the run-time memory layout and binary image layout when direct boot is used for ESP32-C3.
```

Comment:
Is the diagram not valid for H2? If no, could we add a new one or update the existing one to be valid for both chips?

Reply:
No, for the H2 there should be another diagram.
H2 has only one rom, so all sections as: _entry, .text and .rodata are in the same address space. 
Also the address of this rom space is also different. 

@igrr , Is there any chance to get the editable project of this diagram?